src/clj/puppetlabs/services/config/puppet_server_config_service.clj

```diff
@@ -26,7 +26,9 @@
           "Initializing with the following settings from core Puppet:\n%s"
           (ks/pprint-to-string puppet-config))
         (core/init-webserver! override-webserver-settings!
-                              (get-in tk-config [:webserver])
+                              (get-in tk-config
+                                      [:webserver :puppet-server]
+                                      (get-in tk-config [:webserver]))
                               puppet-config)
```

Comment:
@KevinCorcoran We had to do something like that for the legacy-routes-service.  For that case, we used `tk-services/get-service` to get the `:MasterService` and then got its route from that.  So we didn't have to hardcode any namespaces (just the service protocol name) - which was good because the master service's namespace for PE is different from OSS.  But it does still couple the config service to the master service, which isn't great either.

Unless others have a better idea, suppose I'm okay with just hardcoding `:puppet-server` for now.